packages/@okta/vuepress-site/docs/guides/device-authorization-grant/main/index.md

```diff
@@ -10,7 +10,7 @@ layout: Guides
 
 The Device Authorization feature is an OAuth 2.0 grant type. It allows users to sign in to input-constrained devices, such as smart TVs, digital picture frames, and printers, as well as devices with no browser. Device Authorization enables you to use a secondary device, such as a laptop or mobile phone, to complete sign-in to applications that run on such devices.
 
-The Device Authorization feature is available for both Okta Classic and Okta Identity Engine orgs.
+The Device Authorization feature is available for both Okta Classic Engine and Okta Identity Engine orgs.
```

Comment:
I haven't checked lately, but is this called "Okta Classic Engine"?

Reply:
@christinechoy-okta Yes, it's the purpose of this PR :)

The two names are: Okta Identity Engine (or "Identity Engine", for short) and Okta Classic Engine (or "Classic Engine", for short).

Reply:
Are we using the short form here or the longer form?
I notice we're using just "Classic Engine" and "Identity Engine" in the majority of cases here. So should line 13 here be "Classic Engine" and "Identity Engine"?